_layouts/base.html

```diff
@@ -25,7 +25,12 @@
 	<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
 	<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
 	<![endif]-->
-    <title>{{ site.title }}{% if page.title %}: {{ page.title }}{% endif %}</title>
+
+  <title>
+  {% if page.title %}{{ page.title }}: {% endif %}
+  {% if site.title %}{{ site.title }}{% endif %}
```

Comment:
I suggest merging the two lines and displaying `page.title` unconditionally.
```
{% if page.title %}{{ page.title }}: {% endif %}{{ site.title }}
```

Reply:
Assuming you mean "displaying `site.title` unconditionally."

Reply:
yes, site title, sorry for the confusion. Site title must be set (in the `_config.yml`)